plugins/explore-backend/package.json

```diff
@@ -42,6 +43,7 @@
     "supertest": "^6.2.4"
   },
```

Comment:
you'll need to add the `configSchema` key here as well

Reply:
good catch, thanks! Fixed.